changelog/unreleased/pull-3514

```diff
@@ -0,0 +1,6 @@
+Bugfix: timeout error on slow rclone backend
```

Comment:
I'd change this to:
`Enhancement: Support timeout configurable for rclone backend`